src/tests/system/tests/test_passkey.py

```diff
@@ -0,0 +1,495 @@
+from __future__ import annotations
+
+import pytest
+import os
+
+from lib.sssd.roles.client import Client
+from lib.sssd.roles.ipa import IPA
+from lib.sssd.roles.ldap import LDAP
+from lib.sssd.topology import KnownTopology
+
+
+@pytest.mark.tier(0)
+@pytest.mark.topology(KnownTopology.Client)
+def test_passkey__sssctl_register(client: Client):
+    """
+    :title: Register a key with sssctl
+    :setup:
+        1. Setup IDM client with FIDO and umockdev setup
+    :steps:
+        1. Use sssctl to register a FIDO2 key.
+        2. Check umockdev-run that the output contains the user key mapping data.
+    :expectedresults:
+        1. New key is registered
+        2. Output contains key mapping data.
+    :customerscenario: False
+    """
+    tc = "sssctl_reg_test1"
+    cmd = "sssctl passkey-exec --register --username=user1 --domain=ldap.test"
```

Comment:
sssctl passkey-exex --register should be part of passkey.register method, username and domain should be given as parameter to that method

Reply:
but this is on way to register the key,
in second tesr case, we use below command 
cmd = "ipa user-add-passkey user1 --register --cose-type=es256 --require-user-verification=True"

and we need to pass those command to umockdev-run to check the output.